lib/buildsystems/autotools.rb

```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Autotools < Package
+
+  def self.configure_options(options = '')
+    return (@configure_options = options if options)
+  end
+
+  def self.check?(bool = true)
+    return (@check = bool)
+  end
+
+  def self.build
+    # Run autoreconf if necessary
+    unless File.executable? './configure'
+      if File.executable? './autogen.sh'
+        system 'NOCONFIGURE=1 ./autogen.sh --no-configure || NOCONFIGURE=1 ./autogen.sh'
+      elsif File.executable? './bootstrap'
+        system 'NOCONFIGURE=1 ./bootstrap --no-configure || NOCONFIGURE=1 ./bootstrap'
+      else
+        system 'autoreconf -fiv'
+      end
+    end
+    system "./configure #{CREW_OPTIONS} #{@configure_options}"
```

Comment:
I just thought of an idea that might work.  What are your thoughts of using environment variables to override?  For instance, if `CREW_CUSTOM_ENV_OPTIONS` and/or `CREW_CUSTOM_OPTIONS` is set, use those settings instead of `CREW_ENV_OPTIONS` or `CREW_OPTIONS`?